cameras.h

```diff
@@ -115,6 +115,9 @@
 #ifdef HAVE_ZWO_CAMERA
 # define ZWO_ASI
 #endif
+# ifdef HAVE_SVB_CAMERA
+# define SVB_CAMERA
+# endif 
```

Comment:
cosmetic: please match indent style of adjacent code; pls no trailing whitespace
```suggestion
#ifdef HAVE_SVB_CAMERA
# define SVB_CAMERA
#endif
```